oaknational/Oak-Web-Application
# Remove cycle 2 conditionals from curriculum

## Description
Remove cycle 2 conditionals from the curriculum part of the codebase, as it's now launched

## Issue(s)
Fixes `CUR-1027`

## How to test

1. Go to https://deploy-preview-2994--oak-web-application.netlify.thenational.academy
2. Go to the curriculum visualiser/document(docx) 
3. Check cycle 2 features are still enabled and no regressions 
4. Check no cycle 1 features (subject principals exist)

@rightsaidjames you're probably best looking at the code to remind yourself of the areas that between for cycle 1/2... unless if burnt into you brain from all the QA 🧠
